packages/backend/src/open_payments/payment_pointer/model.ts

```diff
@@ -98,12 +105,21 @@ export class PaymentPointer
       authServer
     }
   }
+
+  public isActive() {
+    return this.status === PaymentPointerStatus.Active
+  }
```

Comment:
having both `status` and `isActive()` is a bit redundant. I could remove one of them. Originally I had `isActive` but then added the status when adding the new payment pointer update method.

I'm also just noticing that isActive could probably just be a getter instead of a regular method, like status.

Reply:
Agree with keeping just `isActive`

Reply:
I think this is ultimately about what we want to return in the API, see my reply here: https://github.com/interledger/rafiki/pull/1417#discussion_r1190885248